aopy/data/base.py

```diff
@@ -898,22 +899,29 @@ def map_data2elecandpos(datain, signalpath_table, eleclayout_table, acq_ch_subse
     
     return dataout, acq_ch_position, acq_chs, connected_elecs
 
-def load_chmap(drive_type='ECoG244', acq_ch_subset=None, theta=0, **kwargs):
+def load_chmap(drive_type='ECoG244', acq_ch_subset=None, theta=0, center=(0,0), **kwargs):
     '''
-    Load the mapping between acquisition channel and electrode number for the viventi ECoG array or between a chamber grid and chamber location.
+    Load the centered mapping between acquisition channels and electrode position for supported drives.
+    Currently supports 'ECoG244', 'Opto32', 'NP_Insert72', and 'NP_Insert137' drives.
     
     Args:
-        drive_type (str, optional): Drive type of the method used to record neural activity. Currently supports `ECoG244`, 'Opto32', 'NP_Insert72', and 'NP_Insert137'
+        drive_type (str, optional): Drive type of the method used to record neural activity. 
+            - 'ECoG244': Viventi 244 channel ECoG array          
+            - 'Opto32': Orsborn 32 channel fiber optic array
+            - 'NP_Insert72': Orsborn 72 site Neuropixel grid
+            - 'NP_Insert137': Orsborn 137 site Neuropixel grid
         acq_ch_subset (nacq, optional): Subset of acquisition channels to call. If not called, all acquisition 
             channels and connected electrodes will be returned.
         theta (float): rotation (in degrees) to apply to positions. rotations are applied clockwise, e.g., theta = 90 
             rotates the map clockwise by 90 degrees, -90 rotates the map anti-clockwise by 90 degrees. Default 0.
+        center (2-tuple): chamber coordinates of the center of the drive in mm. Defaults to (0,0).
```

Comment:
This definition seems a little ambiguous. Can you add something like "This function translates the coordinates of the drive to be centered on this value."